src/pip/_internal/commands/hash.py

```diff
@@ -27,7 +29,9 @@ class HashCommand(Command):
     ignore_require_venv = True
 
     def __init__(self, *args, **kw):
-        super(HashCommand, self).__init__(*args, **kw)
+        # type: (List[Any], Dict[Any, Any]) -> None
+        # https://github.com/python/mypy/issues/4335
+        super(HashCommand, self).__init__(*args, **kw)  # type: ignore
```

Comment:
I think, after using `*Any` and `**Any`, you may also remove the `#type: ignore` from here as well.

Reply:
Aah good catch, fixed in latest commit.